### DISCUSSION TITLE
How to discover applications using cppfront?

### DISCUSSION THREAD

**Discussion by parsifal-47** [hsutter/cppfront#1050] [Q&A]
Hi All,
I was curious to see something bigger than helloworld, but what I found so far is only skeleton application with cmake and conan recipes. I tried github search, it found a list of cpp2something repositories and it did not recognize it as a language. I tried google search, it did not help me as well. Is there a list of known cppfront experiments or some other way to discover them?

Thank you!

**Comment by JohelEGP** [comment]
Try <https://github.com/search?q=%28path%3A*.cpp2+AND+%28content%3A%2F%3A+%3F%5B%3D%28%5D%2F+OR+content%3A%2F%5Ba-zA-Z0-9_%5D%5C%24%2F+OR+content%3A%2F%5B-%2B*%26%7E.%29%5D%5C%24%2F%29%29+NOT+is%3Afork&type=code> (from <https://github.com/hsutter/cppfront/issues/459#issuecomment-1809136488>).

**Comment by isidorostsa** [comment]
You might find my [ray-tracing-in-a-weekend C++2 port](https://github.com/isidorostsa/RayTrayCpp2) interesting
